### PR TITLE
Add counts for ignored , measured, filtered

### PR DESCRIPTION
**Pull request by ztplz**

### CODE REVIEW

File: js/testing/testing.ts [hunk 1]
```diff
@@ -41,6 +45,9 @@ export function test(t: TestDefinition | TestFunction): void {
   }
   if (filter(name)) {
     tests.push({ fn, name });
+    filtered++;
```
Comment (kt3k) [inline js/testing/testing.ts]: Is this right place to increment filtered(-out) counts?

**Comment by kt3k** [comment]
It seems rust uses `ignored` and `measured` counts for [benchmark testing](https://doc.rust-lang.org/1.5.0/book/benchmark-tests.html). I'm curious if these counters work in the same way as rust.

File: js/testing/testing.ts [hunk 2]
```diff
@@ -41,6 +45,9 @@ export function test(t: TestDefinition | TestFunction): void {
...
+  } else {
+    ignored++;
```
Comment (ry) [inline js/testing/testing.ts]: filtered++ should be here.

Remove `ignored++` it seems to be for benchmarks.

File: js/testing/testing.ts [hunk 3]
```diff
@@ -71,6 +78,7 @@ async function runTests() {
   console.log("running", tests.length, "tests");
   for (let i = 0; i < tests.length; i++) {
     const { fn, name } = tests[i];
+    measured++;
```
Comment (ry) [inline js/testing/testing.ts]: measured is for benchmarks, remove.

**Pull request closed without merging**